src/collective/easyform/migration/actions.py

```diff
@@ -111,6 +111,14 @@ def actions_model(ploneformgen):
             )
             continue
 
+        # remove outdated/deleted fields from the showfields list for saveddata adapters
+        # otherwise a keyerror will be thrown when viewing the data in easyform
+        if type_.name == 'collective.easyform.actions.SaveData':
+            filteredFields = []
+            available_fields = [field.getName() for field in ploneformgen.fgFields()]
```

Comment:
No need to create a list:
```suggestion
            available_fields = (field.getName() for field in ploneformgen.fgFields())
```

Reply:
I do agree it should be equivalent. However, for the sake of readability, I will keep the list.